airflow/www/static/js/tree/Tree.jsx

```diff
@@ -85,7 +91,10 @@ const Tree = () => {
   }, [tableRef, scrollOnResize]);
 
   return (
-    <Box>
+    <Box mt={3}>
+      <FilterBar />
+      <LegendRow />
+      <Divider mb={5} borderBottomWidth={2} />
       <Flex flexGrow={1} justifyContent="flex-end" alignItems="center">
```

Comment:
Would it make sense to move autorefresh and the sidepanel button into Filter bar? Or is that too crowded?

Reply:
I tried for sanity, on a 13inch this will look close to this (~1300px width), it's quite close to warpping and quite crowded. IMO i would leave it as is, but I am happy to make the change if you want.

![image](https://user-images.githubusercontent.com/14861206/166563808-9c238b53-aa44-438b-b97c-602b2ec78fe6.png)

Reply:
Ok. Leave it as is. We can always change that later.